main.py

```diff
@@ -1,10 +1,37 @@
+import json
 import init_django_orm  # noqa: F401
-
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+    for user_name in data:
+        user = data[user_name]
+        race, flag = Race.objects.get_or_create(
+            name=user["race"]["name"],
+            description=user["race"]["description"])
+
+        for skill_data in user["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                bonus=skill_data["bonus"], race=race
+            )
+
+        if user.get("guild"):
+            guild, flag = Guild.objects.get_or_create(
+                name=user["guild"]["name"],
+                description=user["guild"]["description"])
+        else:
+            guild = None
```

Comment:
You can create this viriable at the start and do not use else block .

If your condition is triggered, the contents of the variable will simply be overwritten